tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -152,13 +149,27 @@ impl Default for Format<Full, SystemTime> {
     }
 }
 
+#[cfg(feature = "json")]
+#[cfg_attr(docsrs, doc(cfg(feature = "json")))]
+impl<T> Format<Json, T> {
+    /// Use the full JSON format with event metadata flattened.
```

Comment:
I think it would be nice if there was an example of the flattened JSON output here? The docs don't make it super clear what "flattened" actually means.

Reply:
(the `json` function or `Json` type should probably have an example of the default output, too, so you can compare and contrast them?)